EIPS/eip-8099.md

```diff
@@ -0,0 +1,165 @@
+---
+eip: 8099
+title: MEVless Protocol
+description: A protocol that prevents MEV attacks by constraining transaction ordering through blind sequencing and two-phase block execution.
+author: Lawliet Chan (@lawliet-chan)
+discussions-to: https://ethereum-magicians.org/t/mevless-an-anti-mev-protocol/26800
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-09-25
+---
+
+## Abstract
+
+This proposal introduces the MEVless protocol, a novel approach to prevent Maximum Extractable Value (MEV) attacks by implementing blind transaction sequencing. The core principle of MEVless is to prevent validators from seeing transaction content during the ordering phase, thereby eliminating the fundamental prerequisite for MEV attacks. The protocol separates block production into two phases: sequencing blocks that order transactions based on transaction hashes without revealing transaction content, and execution blocks that execute transactions in the predetermined order. This design eliminates the ability of miners and validators to perform MEV attacks such as sandwich attacks and front-running by removing their access to transaction content during the critical ordering phase.
+
+## Motivation
+
+MEV attacks, particularly sandwich attacks and front-running, represent a significant threat to blockchain ecosystems, especially in DeFi applications. The fundamental prerequisite for MEV attacks is that validators can see transaction content before ordering, which allows them to:
+
+1. **Sandwich Attacks**: Insert buy and sell transactions around user transactions to manipulate prices and extract value
+2. **Front-running**: Execute similar transactions before users to capture arbitrage opportunities
+3. **Back-running**: Execute transactions after users to benefit from price movements
+
+Unlike legitimate arbitrage activities that provide liquidity, these attacks are purely extractive and harmful to the ecosystem. They reduce user confidence, increase transaction costs, and can lead to significant financial losses for users.
+
+**L2 as the Primary anti-MEV Battlefield in the future**: MEV attacks are particularly problematic on L2 networks, which have become the primary venues for DeFi activities due to lower transaction costs and higher throughput. L2 networks should be the primary focus for MEV resistance, not L1, because L2s can implement more aggressive anti-MEV measures at the protocol level. Unlike L1 which must maintain broad compatibility, L2 networks can be purpose-built for specific applications (DeFi, DEX, etc.) and implement fundamental changes to transaction ordering and execution that would be impossible on general-purpose L1 chains. The specialized nature of L2 applications allows for more thorough MEV resistance through protocol-level modifications.
```

Comment:
```suggestion
```
seems like you need to reframe the proposal for L1, this is giving rationale for L2s

Reply:
pls go through and make sure entire proposal is geared towards L1, you can talk about L2 if L1 needs to provide something to L2s through this proposal